students/15th/team2/seunghee_mun/board/views.py

```diff
@@ -0,0 +1,92 @@
+import json
+from django.http import JsonResponse
+from django.views import View
+from django.core.serializers import serialize
+from board.models import Board, Comment
+from user.models import User
+
+
+class BoardView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        # login_user validation
+        try:
+            user_post = User.objects.get(user_name=data['user']).id
+        except:
+            return JsonResponse({'MESSAGE' : 'NOT_MEMBER'}, status=400)
```

Comment:
- `try`...`except` 문을 잘 사용하셨지만 에러는 명확하게 잡아주는게 좋습니다.
- 유저 테이블에 조건에 맞는 유저정보가 존재하지 않을 시 `DoesNotExist` 에러를 raise 하게 됩니다. 따라서 아래처럼 수정해주실 수 있습니다.
```suggestion
        try:
            user_post = User.objects.get(user_name=data['user']).id
        except User.DoesNotExist:
            return JsonResponse({'MESSAGE' : 'NOT_MEMBER'}, status=400)
```

Reply:
try, except 문에 정확한 에러를 지정해줘하는게 정말 중요하다는것을 알았습니다.
Django의 get method exception 은 총 3가지가 있으며 해당 로직에서는 값이 없을때 나오는 에러만 반환하도록 지정해주겠습니다.
다른것도 모두 이와 같이 수정하겠습니다.